v2/tools/generator/internal/functions/kubernetes_admissions_validations.go

```diff
@@ -92,6 +101,42 @@ func validateResourceReferencesBody(codeGenerationContext *astmodel.CodeGenerati
 	return body
 }
 
+func validateOwnerReferences(k *ResourceFunction, codeGenerationContext *astmodel.CodeGenerationContext, receiver astmodel.TypeName, methodName string) *dst.FuncDecl {
+	receiverIdent := k.IdFactory().CreateReceiver(receiver.Name())
+	receiverType := receiver.AsType(codeGenerationContext)
+
+	fn := &astbuilder.FuncDetails{
+		Name:          methodName,
+		ReceiverIdent: receiverIdent,
+		ReceiverType:  astbuilder.PointerTo(receiverType),
+		Body:          validateOwnerReferencesBody(codeGenerationContext, receiverIdent),
+	}
+
+	fn.AddReturn(astbuilder.QualifiedTypeName(codeGenerationContext.MustGetImportedPackageName(astmodel.ControllerRuntimeAdmission), "Warnings"))
```

Comment:
Prefer putting the call to `MustGetImportedPackageName()` onto a separate line assigning a new variable, largely so any stack trace directly isolates the relevant call.